maplibre_gl/android/src/main/java/org/maplibre/maplibregl/GlobalMethodHandler.java

```diff
@@ -79,8 +79,8 @@ public void onMethodCall(MethodCall methodCall, MethodChannel.Result result) {
         result.success(null);
         break;
       case "setOffline":
-        boolean offline = methodCall.argument("offline");
-        ConnectivityReceiver.instance(context).setConnected(offline ? false : null);
+        Boolean offline = (Boolean) methodCall.argument("offline");
+        ConnectivityReceiver.instance(context).setConnected(offline!=null ? !(offline) : null);
```

Comment:
Please use `!offline.booleanValue()` to make the conversion explicit. Also, please add spaces around `!=`.